team_test.go

```diff
@@ -69,14 +69,250 @@ func TestGetTeamWithAlias(t *testing.T) {
 
 func TestListTeams(t *testing.T) {
 	// Arrange
-	client := ATestClient(t, "team/list")
+	requests := []TestRequest{
+		{`{
+    "query": "query ($after:String!$first:Int!){account{teams(after: $after, first: $first){nodes{alias,id,aliases,contacts{address,displayName,id,type},group{alias,id},htmlUrl,manager{id,email,htmlUrl,name,role},members{nodes{id,email,htmlUrl,name,role},pageInfo{hasNextPage,hasPreviousPage,startCursor,endCursor}},name,responsibilities,tags{nodes{id,key,value},pageInfo{hasNextPage,hasPreviousPage,startCursor,endCursor},totalCount}},pageInfo{hasNextPage,hasPreviousPage,startCursor,endCursor}}}}",
+    "variables": {
+      "after": "",
+      "first": 100
+    }
+  }`,
+			`{
+    "data": {
+      "account": {
+        "teams": {
+          "nodes": [
+            {
+              "alias": "devops",
+              "aliases": [
+                "devops"
+              ],
+              "contacts": [],
+              "htmlUrl": "https://app.opslevel.com/teams/devops",
+              "id": "Z2lkOi8vb3BzbGV2ZWwvVGVhbS83NzQ",
+              "manager": {
```

Comment:
As extra credit you could turn some of these things in reusable templates

If you look in `./templates/...` you can see how we define the templates - for example maybe you could make `./templates/user.tpl` and then define

```
{{- define "user_id_1" }}
{
"name": "Kyle Rockman",
"email": "kyle.rockman@mac.com"
}
{{ end }}
```

and then you can convert this to

```suggestion
              "manager": {{ template "user_id_1" }},
```